eventing/README.md

```diff
@@ -18,143 +18,171 @@ The above concerns are consistent with the
 a common specification for cross-service interoperability being developed by the
 CNCF Serverless WG.
 
-> **Note**: The Eventing repo is very much a work-in-progress. See the
-> [list of Known Issues](https://github.com/knative/eventing/issues?q=is%3Aissue+is%3Aopen+label%3A%22Known+Issue%22)
-> with the current state, but feel free to report additional issues. Expect to
-> see changes in all of the following areas in the next months:
->
-> 1.  The API model and terminology is still evolving.
-> 1.  The controller model for Buses and Sources are still being refined.
-> 1.  Test coverage is not up to snuff.
-> 1.  The release packaging is still being determined.
-
-Once these concerns are resolved, we expect to work on creating a wider array of
-EventSources.
+To enable delivery to multiple types of Services, Knative Eventing defines two
+generic interfaces which multiple Kubernetes resources can implement:
+
+1. **Addressable** objects are able to receive and acknowledge an event
+   delivered over HTTP to an address defined in their `status.address.hostname`
+   field. As a special case, the core
+   [Kubernetes Service object](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#service-v1-core)
+   also fulfils the Addressable interface.
+2. **Callable** objects are able to receive an event delivered over HTTP and
+   transform the event, returning 0 or 1 new events in the HTTP response. These
+   returned events may be further processed in the same way that events from an
+   external event source are processed.
+
+Knative Eventing also defines a single event forwarding and persistence layer,
+called a **Channel**. Messaging implementations may provide implementations of
+Channels via the ClusterChannelProvisioner object. This allows message delivery
```

Comment:
already merged but I would have put a URL for the Channels and ClusterChannelProvisioner specs